builtin/credential/aws-kms/path_login.go

```diff
@@ -0,0 +1,128 @@
+package awsKms
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathLogin(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "login",
+		Fields: map[string]*framework.FieldSchema{
+			"key_id": &framework.FieldSchema{
```

Comment:
What is this for, Is this a left over artifact from when you where using the key id from the login request instead of the AWS decrypt response?